### PR TITLE
make search box in issue sidebar dropdown list always show when scrolling

### PR DESCRIPTION
**Pull request by a1012112796**
as title, replace #31597 after #32460, examples:

image

----
image
----
屏幕截图 2024-11-20 181852
---

### CODE REVIEW

File: web_src/css/repo.css [hunk 1]
```diff
@@ -66,6 +66,12 @@
   overflow-x: auto;
 }
 
+.issue-content-right .dropdown > .menu .item-secondary-info small {
```
Comment (wxiaoguang) [inline web_src/css/repo.css:69]: Could the style be on `.issue-content-right .dropdown > .menu .item-secondary-info` instead of `small`?

**Review by wxiaoguang — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by wxiaoguang** [comment]
The scrolling menu has border, also need this:  Improve issue sidebar UI #32587

**Comment by wxiaoguang** [comment]
Need to fix:  Fix issue sidebar regression #32598

**Comment by wxiaoguang** [comment]
It seems that there is another regression, the dropdown loses keyboard support

ArrowUp/ArrowDown/Enter doesn't work.

**Pull request merged**